src/cppcache/test/ByteArray.cpp

```diff
@@ -20,7 +20,7 @@
 #include "config.h"
 
 #ifdef _MACOSX
-#include <stdlib.h>  // For wcstombs()
+#include <cstdlib>  // For wcstombs()
 #endif
 #include <wchar.h>
```

Comment:
I hadn't worried as much about these ones because they weren't in the public headers *and* part of test.  But I'll try deleting and see what happens.